MhLabs.AwsSignedHttpClient/BaseHttpMessageHandler.cs

```diff
@@ -21,6 +21,7 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             if (!request.Headers.Contains(CorrelationHelper.CorrelationIdHeader))
             {
                 request.Headers.Add(CorrelationHelper.CorrelationIdHeader, CorrelationHelper.CorrelationId ?? Guid.NewGuid().ToString());
+                request.Headers.Add("X-Amzn-Trace-Id", Environment.GetEnvironmentVariable("_X_AMZN_TRACE_ID"));
```

Comment:
Only set header if the environment variable contains a value

Reply:
Fixed!